functions-and-scope.js

```diff
@@ -72,6 +141,19 @@ const grades = [9, 8, 5, 7, 7, 4, 9, 8, 8, 3, 6, 8, 5, 6];
 
 // ---- Verwachte uitkomst: 9
 
+const largestNumber = (number) => {
+    let highestGrade = 0;
```

Comment:
In deze opdracht is het prima om als default waarde 0 te pakken, maar stel je eens voor dat hier een array binnenkomt als [-1, -4, -2, -3]. In dit voorbeeld is de hoogste waarde -1, maar met jouw functie zou er 0 uit komen terwijl die niet eens in de array voorkomt.

Nogmaals, het maakt voor deze situatie niet uit, maar het is goed om je te realiseren dat wanneer operaties als deze gaat doen op een array, dat je niet een default waarde pakt, maar altijd de eerste waarde van de array, dan weet je zeker dat je goed zit namelijk:
```suggestion
    let highestGrade = number[0];
```